tests/test_server.py

```diff
@@ -52,140 +71,152 @@ def test_add_device(client, name='steelseries-rival310'):
     data = {
         'shortname': name
     }
-    response = client.post('/devices/add',
-                           data=json.dumps(data),
-                           content_type='application/json')
+    response = client.post('/devices/add', json=data)
     assert response.status_code == 201
-    data = json.loads(response.data)
+    data = response.json()
```

Comment:
`data` is hiding the json `data` description of the device

Reply:
Yes, to avoid constantly writing `response.json()` in the asserts bellow.

Reply:
I understand why you need a variable, I am complaining that you are hiding a previous variable. Rename this one to `answer` or something else, please.